api/files/views.py

```diff
@@ -59,7 +301,47 @@ def get_object(self):
 
 
 class FileVersionsList(generics.ListAPIView, FileMixin):
-    """List of versions for the file requested.
+    """List of versions for the requested file. *Read-only*.
+
+    Paginated list of file versions, ordered by increasing version number (id).
+
+    <!--- Copied Spiel from FileVersionDetail -->
+
+    A specific version of an uploaded file.  Note that the version is tied to the id/path, so two versions of the same
+    file could have completely different contents and formats.  That's on you, though.  Don't do that.
+
+    Unlike the OSF File entity which can represent files and folders, FileVersions only ever represent files. When a
+    file is first uploaded through the API it is assigned version 1.  Each time it is updated through the API, the
```

Comment:
This is only true for osfstorage, version ids are otherwise pulled from their provider